CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Contributing
+
+Thank you for taking time to contribute (or at least considering to). CMS-Android is a community project as much as
+it is Crux's. 
+
+Contributions does not necessarily mean you have to modify source code and fix a bug or add a new functionality.
+Reporting a bug using the _Issues_ tab is also highly appreciated and a good way to start contributing to a project. 
+start off contributions.
+
+It is essential that you know how to use Git as well as GitHub. While the maintainers will be more than happy helping
+you make your first PR, prerequisite knowhow makes it easier for everyone involved. Check this 
+[guide](https://github.com/firstcontributions/first-contributions)  if you've never contributed to a project on 
+GitHub before.
+
+Please note we have a [code of conduct](code-of-conduct.md), please follow it in all your interactions with the project.
+
+When you submit code changes, your submissions are understood to be under the same [MIT License](LICENSE) that covers the 
+project. Feel free to contact the maintainers if that's a concern.
+
+## Moodle Endpoints Used By The App
+
+The Moodle endpoints used by this app have been collected in a [Postman](https://getpostman.com") workspace 
+[Here](https://www.getpostman.com/collections/e2c0439f144f7d3f60ed). You may import this collection into Postman directly using the link. 
+
+## Contributors Pull Request Checklist
+
+1. Fork the repository and branch from `development`.
+
+2. Work on your changes. Make sure you commit as frequently as required. Once you're done making the changes, push the commits
+   to your fork. Checkout the **Commit Messages** section to find guidelines relating to commit messages.
+
+3. If the changes you make fixes a bug in `master` or adds a new feature, note it down in the [changelog](CHANGELOD.md).
+   
+3. Head over to the [pull requests](https://github.com/crux-bphc/CMS-Android/pulls) page and create a new pull request. Make sure that the PR
+   is from your feature branch into this repo's `development` branch. If you are making changes that affect the UI in anyway, attach an image
+   in your PR description.
+
+4. Wait for a mainter to review your commit. If they finds any issue (including redundant code, unnecessary changes, potential bugs),
+   they will point it out to you. Make any necessary changes and push them to your fork. Ideally, try not to force push as it becomes
+   harded for the maintiners to keep track of changes. Mark the review as resolved.
+
+5. If you are a maintainer yourself, try to get a secondary opinion from other maintainers or contributors.
+
+6. If everything looks okay, the mainter(s) will go ahead and merge your changes into the repo. 
+
+**Note for maintainers**: When merging a PR in, rebase the changes instead of creating a merge commit. This ensures a clean commit history.
```

Comment:
Can `rebase the changes instead of creating a merge commit' be highlighted a bit`? 
example bold